packages/next/build/webpack/loaders/babel-loader/src/cache.js

```diff
@@ -47,12 +46,9 @@ export default async function handleCache(params) {
 
     // Otherwise just transform the file
     // return it to the user asap and write it in cache
```

Comment:
I don't have an opinion one way or another, although it would be a behavior changes distinct from what else is in the PR.  Would it be alright to fix separately after this PR merges?  I don't mind doing that.

Reply:
Yeah can be changed in a different PR if we want 👍